### DISCUSSION TITLE
Two thoughts on Tesla

### DISCUSSION THREAD

**Discussion by tomfid** [Data4DM/BayesSD#146] [people; relating]
Ironically the two are in conflict because the first (product vs process improvement tradeoffs speculative model) conceptualizes process improvement as defect reduction, which the second (Ackoff on systems) points out is the wrong perspective. 

1.

[product process 2bb.mdl.zip](https://github.com/Data4DM/BayesSD/files/11194590/product.process.2bb.mdl.zip)

2.

https://youtu.be/OqEeIG8aPPk

**Comment by hyunjimoon** [comment]
The vensim model presents worse and better dynamics:
<img width="274" alt="image" src="https://user-images.githubusercontent.com/30194633/232163851-901b6226-d847-4d3f-b919-bbf85ca67b4b.png">

[need to review zoom recording]
- preassembling can test (not just pieces but interdependency - defect earilier, important pj)
- design and instruction and deployment of the field
- product defect (cause pd to later), 
- bad design likely to fail (production processes)
- how much defects to catch (reporting defects eariler, discovery rates are highter but not )
- vensim's new  policy outcome is # of defect - what to do to improve that? do a better jobs and report error sensitivity analysis (as output of goal - organize result) <img width="1200" alt="image" src="https://user-images.githubusercontent.com/30194633/232164565-5228b4a7-7a3b-4c9b-a35b-bd234d57cf5d.png">
